src/pages/EditMonitor.vue

```diff
@@ -899,6 +902,28 @@
                                     </div>
                                 </template>
                             </template>
+
+                            <!-- TLS Options -->
+                            <template v-if="monitor.type === 'port-tls'">
+                                <h2 class="mt-5 mb-2">{{ $t("TLS Options") }}</h2>
+
+                                <!-- Use STARTTLS -->
+                                <div class="my-3 form-check">
+                                    <input id="start-tls" v-model="monitor.startTls" class="form-check-input" type="checkbox">
+                                    <label class="form-check-label" for="start-tls">
+                                        {{ $t("Use STARTTLS") }}
```

Comment:
I think this might be a good case for linking to the docs at https://datatracker.ietf.org/doc/html/rfc3207

What do you think? (Tip: [Componentised translations via `i18n-t`](https://vue-i18n.intlify.dev/guide/advanced/component) might be benefitial for this)

Reply:
STARTTLS is pretty common and shows up in every email program. But a link to the RFC can't hurt, so let me add that. Thanks for the translation tip there, that looks useful indeed.